apps/docs/src/index.data.ts

```diff
@@ -63,78 +62,90 @@ export default defineLoader({
      * Also returns a `href` property with the link to the implemented component (only if implemented).
      */
     const getImplementedStatus = (componentName: string) => {
-      const implemented = watchedFiles.some((file) => file.endsWith(`${componentName}.stories.ts`));
+      const fileExist = watchedFiles.some((file) => file.endsWith(`${componentName}.stories.ts`));
       return {
-        implemented,
-        href: implemented ? `/development/components/${componentName}` : undefined,
-      };
+        status: fileExist ? "in-progress" : "planned",
+        href: fileExist ? `/development/components/${componentName}` : undefined,
+      } satisfies Partial<ComponentCardProps>;
     };
 
-    const componentTabs: HomePageData["componentTabs"] = [
+    const basicComponentsDueDate = new Date(2024, 4);
+
+    const components: HomePageData["components"] = [
+      {
+        name: "Button",
+        dueDate: basicComponentsDueDate,
+        ...getImplementedStatus("OnyxButton"),
```

Comment:
question: for the third status, instead of using the function `getImplementedStatus`, would we write `implemented` by hand?

Reply:
Thats how I intended it at least since we agreed that we set them manually to "implemented"